PyrusClient.cs

```diff
@@ -21,6 +21,17 @@ public class PyrusClient
 
 		public static Settings Settings { get; set; }
 
+		/// <summary>
+		/// Returns custom settings if they were specified in <see cref="PyrusClient"/> constructor.
+		/// Otherwise, returns static settings
+		/// </summary>
+		public Settings CustomSettings 
```

Comment:
Let's do it like this
```c#
public Settings ClientSettings { get; private set; }
```

Reply:
renamed property, we can make it without a setter, removed it